src/Parser/Hunter/BeastMastery/Modules/Spells/AzeriteTraits/HazeOfRage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import {formatNumber, formatPercentage} from 'common/format';
+import {calculateAzeriteEffects} from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'Interface/Others/TraitStatisticBox';
+
+
+const hazeOfRageStats = traits => Object.values(traits).reduce((obj, rank) => {
```

Comment:
As there's only one stat for Haze of Rage, if you wanted this could be simplified slightly:
```
const hazeOfRageAgility = traits => Object.values(traits).reduce((total, rank) => {
  const [agility] = calculateAzeriteEffects(SPELLS.HAZE_OF_RAGE.id, rank);
  return total + agility;
}, 0);
```
That would mean you no longer need to extract the `agility` prop from the object returned by `hazeOfRageStats` as it would directly return the agility value.